Backend/app/rider-platform/rider-app/Main/spec/Storage/sos.yaml

```
@@ -73,6 +80,17 @@ Sos:
     findByPersonId:
       kvFunction: findAllWithKV
       where: personId
+    updateExternalReferenceId:
```

YAMLlint (1.38.0)

[error] duplication of key "updateExternalReferenceId" in mapping

(key-duplicates)